README.md

```diff
@@ -69,108 +69,116 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Terminal plugin performance boost.  Thanks [@htcfreek](https://github.com/htcfreek)!
-- Terminal will now be found via the Program plugin again.  
-- Shutdown command is now using hybrid fast argument for shutting down
-- Support for VSCodium with VS Code workplace plugin. Thanks [@makeProjectGreatAgain](https://github.com/makeProjectGreatAgain)
+- New plugin for time and date values/information. Thanks [@htcfreek](https://github.com/htcfreek)!
+- WindowWalker now has a tooltip, additional features like killing the process and closing the window, additional settings and improvements. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Unit converter now accepts additional ways of writing feet and gallon. Also interprets as either imperial or US gallons depending on current culture. Thanks [@yifan-k](https://github.com/yifan-k)!
+- Unit converter now accepts metre for meter.
+- Localization for Web Search and Unit Converter (not including units) has been enabled.
+- Localization for Windows Terminal has been enabled. (This was a hotfix for 0.56)
+- Calculator now tries to always interpret the dot (.) symbol as a decimal separator, despite configured culture, to meet expectations.
+- Calculator now handles trailing zeroes on hexadecimal numbers correctly.
+- System commands plugin can now show the local ip and mac addresses. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Folder plugin has improved results, with improved tooltips. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Windows settings plugin has added entries for Screen Saver and Connect Wiring Display Panel. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Plugins can now show descriptions for their configurations in settings. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Fix for the focus issue when calling PowerToys Run for the first time after login and after returning from some windows.
+- Fix for a bug on Program when creating a shortcut.
+- Validated that upgrading to .net framework 6 fixed the error appearing when shutting down the system with PowerToys Run running.
 
 ### Video conference mute
 
-- nVidia Broadcast software won't crash anymore
+- Newly added microphones are now updated and tracked by VCM.
 
 ### Settings
 
-- Upgrade notes in OOBE
-- Fix for settings being lost (This was a hotfix for 0.55)
-- UX improvements. Thanks [@niels9001](https://github.com/niels9001)
+- What's new button in the bottom with a new look, with a couple more UI tweaks. Thanks [@niels9001](https://github.com/niels9001)!
+- Fixed a bug causing Settings not to open when a racing condition caused Keyboard Manager settings to not be read correctly.
+- To open settings from the tray icon a double click is needed instead of a single click.
+- Fix for a bug which would cause checking for updates to run indefinitely.
+- When auto-updating, pass a flag to avoid restarting without being prompted.
```

Comment:
Restarting PT or computer?

Reply:
good catch. this is about rebooting the computer